CMakeLists.txt

```diff
@@ -23,9 +23,14 @@ if(NOT BUILD_LIBSYSTEMD)
         message(WARNING "libsystemd not found, checking for libelogind instead")
         pkg_check_modules(Systemd IMPORTED_TARGET GLOBAL libelogind>=236)
         if(TARGET PkgConfig::Systemd)
-            set(LIBSYSTEMD "libelogind")
+            set(LIBSYSTEMD "elogind")
             string(REPLACE "." ";" VERSION_LIST ${Systemd_VERSION})
             list(GET VERSION_LIST 0 Systemd_VERSION)
+	else()
+            pkg_check_modules(Systemd IMPORTED_TARGET GLOBAL basu)
```

Comment:
I suggest adding a short comment here about `basu` (what it is) and FreeBSD support, and also add `message(WARNING "libelogind not found, checking for basu instead")` like above in the file

Reply:
- explaining `basu` wouldn't be consistent with `elogind`
- added (squashed) `message(...)` as suggested

Reply:
FWIW, `basu` support isn't FreeBSD-specific. Many Linux distros have it packaged, see https://repology.org/project/basu/versions